src/core/text/Text.js

```diff
@@ -103,6 +104,13 @@ export default class Text extends Sprite
         this.style = style;
 
         this.localStyleID = -1;
+
+        /**
+         * Trim transparent borders
+         *
+         * @member {boolean}
+         */
+        this.trim = trim;
```

Comment:
How about `this.trim = trim || false;` or set `trim = false` as the default param to make sure defaults to an actually bool and not undefined.